documentation/user_docs/docs/user_guide/Interfacing_with_other_programs.rst

```diff
@@ -407,11 +406,11 @@ In each case, two elements of the ``DM`` matrix should be varied together, which
 
 In addition to ``mat`` and ``selector``, ``horace_sqw`` also takes some other arguments:
 
-- ``'usefast'`` - This tells ``horace_sqw`` to use a faster but slightly less accurate code than ``spinwave``. In particular, this code achieves a speed gain by:
+- ``'usefast'`` - This tells ``horace_sqw`` to use a faster, more memory efficient but slightly less accurate code than ``spinwave``. In particular, this code achieves its speed gain and lower memory usage by:
 
-    * Only calculating ``Sperp`` rather than the full :math:`S^{\alpha\beta}` tensor
+    * Only calculating the perpendicular component ``Sperp`` rather than the full :math:`S^{\alpha\beta}`
```

Comment:
```suggestion
    * Only calculating the perpendicular component :math:`S_{\text{perp}}` rather than the full :math:`S^{\alpha\beta}`
```
Or would this actually be:
```suggestion
    * Only calculating the perpendicular component :math:`S_{\alpha\alpha}` rather than the full :math:`S^{\alpha\beta}`
```

Reply:
No S_perp is different from the diagonal elements of the tensor.